src/emucore/Settings.cxx

```diff
@@ -300,6 +300,8 @@ void Settings::usage() {
        "     Phosphor blends screens to reduce flicker\n"
        "   -record_screen_dir [save_directory]\n"
        "     Saves game screen images to save_directory\n"
+       "   -stochasticity (default: 0.25)\n"
```

Comment:
Can we name this 'repeat_action_probability' instead? We know what "stochasticity" means because we've been talking about it, but it will likely be very cryptic for anyone else.